kratos/utilities/atomic_utilities.h

```diff
@@ -65,39 +65,6 @@ inline void AtomicSub(TDataType& target, const TDataType& value ) {
     target -= value;
 }
 
-/** @param target vector variable being atomically updated by doing target -= value
- * @param value vector value being subtracted
- * Note that the update is not really atomic, but rather is done component by component
- */
-template<class TVectorType1, class TVectorType2>
-inline void AtomicSub(TVectorType1& target, const TVectorType2& value ) {
```

Comment:
is deleting this method intentional?

Reply:
atomic sub should remain...what is the problem with this?